components/homme/src/theta-l/share/prim_advance_mod.F90

```diff
@@ -789,15 +772,13 @@ subroutine advance_hypervis(elem,hvcoord,hybrid,deriv,nt,nets,nete,dt2,eta_ave_w
   enddo
 
 
-
-
-
 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
   !  sponge layer
 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
   if (nu_top>0 .and. hypervis_subcycle_tom>0 ) then
   dt=dt2/hypervis_subcycle_tom
   do ic=1,hypervis_subcycle_tom
+
```

Comment:
While you're at it, would you mind fixing the indentation of the if block? It should have one extra indentation, but right now it is flushed.

Reply:
I tried to not to touch F code in this PR, unless you really insist.

Reply:
I just thought "while you're at it...".  But I don't really insist.